group.lisp

```diff
@@ -179,9 +187,48 @@ at 0. Return a netwm compliant group id."
       (xlib:change-property (screen-root screen) :_NET_CURRENT_DESKTOP
                             (list (netwm-group-id new-group))
                             :cardinal 32)
+      (mapc (lambda (w)
+              (xwin-unhide (window-xwin w) (window-parent w)))
+            *always-show-windows*)
       (update-all-mode-lines)
       (run-hook-with-args *focus-group-hook* new-group old-group))))
 
+(defun copy-window-to-group (group window)
+  (setf (window-number window) (find-free-window-number group))
+  (push window (group-windows group))
+  (group-add-window group window))
+
+(defun always-show-window (window screen)
+  (let ((groups-to-add-w-to (remove (current-group) (screen-groups screen))))
+    (mapc (lambda (group) (copy-window-to-group group window))
+          groups-to-add-w-to))
```

Comment:
`other-groups` (or `other-groups-in-screen` if one feels like the extra specificity is worth the verbosity) would be a better name than `groups-to-a-w-to`. Also we can use alexandria's rcurry to avoid the lambda clutter. Also `(map nil ...)` communicates that we only care about the side-effects more clearly.

```
(map nil (alexandria:rcurry #'copy-window-to-group window)
      other-groups))
```

or because the variable is only used in one place we can do w/o the let.

```
(map nil (alexandria:rcurry #'copy-window-to-group window)
      (remove (current-group) (screen-groups screen)))
```


(mapc (alexandria:rcurry #'copy-window-to-group window)
      (remove (current-group) (screen-groups screen)))